pkg/gardenlet/apis/config/types.go

```diff
@@ -67,6 +67,10 @@ type GardenletConfiguration struct {
 	// SNI contains an optional configuration for the APIServerSNI feature used
 	// by the Gardenlet in the seed clusters.
 	SNI *SNI
+	// ETCDConfig contains an optional configuration for the
+	// backup compaction feature in etcdbr
```

Comment:
What is etcdbr? Please write proper doc strings here

Reply:
Done